pkg/cinderapi/statefuleset.go

```diff
@@ -113,6 +101,7 @@ func StatefulSet(
 			volumeMounts = append(volumeMounts, svc.CreateVolumeMounts(endpt.String())...)
 		}
 	}
+	readinessProbe.HTTPGet = livenessProbe.HTTPGet
```

Comment:
-1: This line is unnecessary, we already set it on L66

Reply:
fixed